server/channels/utils/subpath.go

```diff
@@ -110,13 +124,29 @@ func UpdateAssetsSubpathInDir(subpath, directory string) error {
 		newRootHTML = strings.Replace(newRootHTML, "</style>", fmt.Sprintf("</style><script>%s</script>", script), 1)
 	}
 
+	if newRootHTML == oldRootHTML {
+		mlog.Debug("No need to rewrite unmodified root.html", mlog.String("from_subpath", pathToReplace), mlog.String("to_subpath", newPath))
```

Comment:
Would it make sense to just log nothing in this case? (Unix philosophy and all?)

Reply:
Oh, I didn't know that was part of the Unix philosophy :eyes: But given this is a debug-level log, I'm in favour of keeping it. I don't think there's such thing as a lot of debug logging! If you have a stronger opinion, I can change it.

Reply:
I'm only 0/5, but if we DEBUG logged everything that /didn't/ happen, I would wager that would be a /lot/ of debug logging ;P. My hope whenever scrubbing logs is to look for positive evidence of something, and in that context, fewer logs that speak to something that actually happened are strictly more helpful. But this is a one-off, so I'm really only -1/5 :D